project/plugins.sbt

```diff
@@ -0,0 +1,6 @@
+addSbtPlugin("org.tpolecat"      % "tut-plugin"   % "0.4.0")
+// addSbtPlugin("com.eed3si9n"      % "sbt-unidoc"   % "0.3.1")
+// addSbtPlugin("com.jsuereth"      % "sbt-pgp"      % "1.0.0")
+// addSbtPlugin("com.github.gseitz" % "sbt-release"  % "1.0.0")
+// addSbtPlugin("org.xerial.sbt"    % "sbt-sonatype" % "0.5.0")
+// addSbtPlugin("com.typesafe.sbt"  % "sbt-osgi"     % "0.8.0")
```

Comment:
You'll probably need all these at some point so I just copied the whole list from doobie and commented them out.